### DISCUSSION TITLE
Carbon emission units

### DISCUSSION THREAD

**Discussion by Ducki** [angularsen/UnitsNet#1361] [Q&A]
I am struggling to use this library with carbon emissions.
For example, a solarfarm has a carbon emission intensity of 42kg per kilowatt hour (kWh as the unit of comparison).
See https://en.wikipedia.org/wiki/Emission_intensity:
`One commonly used figure is carbon intensity per kilowatt-hour (CIPK), which is used to compare emissions from different sources of electrical power.`

How would you define this using this library?

**Comment by angularsen** [comment]
Hi, we don't have any emission intensity units per today. 
This sounds a bit too domain specific to add, UnitsNet aims to cover the most commonly used units.

You can look into custom units and see if that maybe adds some value to your use case:
https://github.com/angularsen/UnitsNet/wiki/Extending-with-Custom-Units
